Automattic/wp-calypso
# Reader: move post-key util to client/reader

This patch just moves the `feed-stream-store/post-key` util to `client/reader`. It is more general purpose than its current location would indicate.  This is split off of streams reduxification in https://github.com/Automattic/wp-calypso/pull/22814

**to test**
- a webpack build should quickly tell you if the imports were messed up
- grep for `lib/feed-stream-store/post-key`, there should be no results.  
- load up the calypso.live branch and make sure Reader still functions correctly without errors in the console
